content/6_Advanced/Max_Flow.mdx

```diff
@@ -14,19 +14,23 @@ frequency: 1
 
 ### Edmonds-Karp Algorithm
 
-The [Edmonds-Karp](https://en.wikipedia.org/wiki/Edmonds%E2%80%93Karp_algorithm) algorithm uses a greedy approach to solve the maximum flow problem.
+The [Edmonds-Karp](https://brilliant.org/wiki/edmonds-karp-algorithm/) algorithm uses a greedy approach to solve the maximum flow problem.
```

Comment:
sus

Reply:
What is sus about this? I have provided an explanation in the commit message that Brilliant Link provides a run time analysis, so I would suggest to link it in the document.

Reply:
damn bro actually writes good commit messages (we don't do those, nor do we read them here)
no what's sus about this is that you removed the wikipedia link, which is fine
at most just link both like (oh yeah here's a link to the brilliant one asw)

Reply:
I edited it again. I kept the Wikipedia link, added a small text, linked the Brilliant Link for the Time Complexity Analysis. The code is still the same as yesterday, played with the formatting of the code a little.